plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -46,18 +46,32 @@ jQuery( function( $ ) {
 		} )
 		// Star ratings for comments
 		.on( 'init', '#rating', function() {
+			$( 'label[for="rating"]' ).hide();
 			$( '#rating' )
 				.hide()
 				.before(
-					'<p class="stars">\
-						<span>\
-							<a class="star-1" href="#">1</a>\
-							<a class="star-2" href="#">2</a>\
-							<a class="star-3" href="#">3</a>\
-							<a class="star-4" href="#">4</a>\
-							<a class="star-5" href="#">5</a>\
-						</span>\
-					</p>'
+					'<fieldset>\
```

Comment:
The rating stars behave now like a radio group. To mitigate regressions on other themes and plugins, I preserved the old markup:
```html
<p>
  <span>
    <a>...</a>
    <a>...</a>
    ...
  </span>
</p>
```

and only wrapped it with `fieldset` and added the `legend`.